docs/api/execute-test.md

```diff
@@ -5,41 +5,98 @@ parent: Benchmark API
 nav_order: 10
 ---
 
-A brief description of the command's function, and how it relates to the Benchmark workflow.
+The `execute_test` command of OpenSearch Benchmark executes tests against your OpenSearch cluster.
 
 
 ## Syntax 
 
 ```bash
-execute_test
+opensearch-benchmark execute_test <arguments>
 ```
 
-// Table for any related fields. Example pulled from PPL page.
 
-Field | Description | Required
-:--- | :--- |:---
-`search` | Specify search keywords. | Yes
-`index` | Specify which index to query from. | No
-`bool-expression` | Specify an expression that evaluates to a boolean value. | No
+### Common Arguments
+
+See [All Arguments](#all-arguments) for an exhaustive list of arguments
 
+Argument | Description | Required
+:--- | :--- |:---
+`workload` | The dataset and operations that execute during a test. See [OpenSearch Benchmark Workloads repository](https://github.com/opensearch-project/opensearch-benchmark-workloads) for more details on workloads. | Yes
+`workload-params` | Parameters defined within each workload that can be overwritten, defined in the README of each workload. You can find an example of the eventdata workload [here](https://github.com/opensearch-project/opensearch-benchmark-workloads/tree/main/eventdata#parameters).  | No
+`test_procedure` | Test Procedures define the sequence of operations and parameters for a specific workload. When there is `test_procedure` specified, Benchmark selects the default. Define test procedures within each workload directory. You can fine an example test procedure [here](https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/main/eventdata/test_procedures/default.json). | No
```

Comment:
1. Nit: "When there is `test_procedure` specified" --> "When `test_procedure` specified". Also regarding this sentence, I think it should be "When there is no `test_procedure` specified, Benchmark selects the default"? 
3. Spelling error:  **fine**--> **find** in the last sentence